src/common/types/store-data/card-view-model.ts

```diff
@@ -0,0 +1,15 @@
+import { GuidanceLink } from '../../../scanner/rule-to-links-mappings';
```

Comment:
No copyright header.

Reply:
Yep! Noticed it earlier; it should be there now